x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/signals_utility_bar/index.tsx

```diff
@@ -33,103 +33,100 @@ interface SignalsUtilityBarProps {
   updateSignalsStatus: UpdateSignalsStatus;
 }
 
-export const SignalsUtilityBar = React.memo<SignalsUtilityBarProps>(
-  ({
-    areEventsLoading,
-    clearSelection,
-    totalCount,
-    selectedEventIds,
-    isFilteredToOpen,
-    selectAll,
-    showClearSelection,
-    updateSignalsStatus,
-    sendSignalsToTimeline,
-  }) => {
-    const [defaultNumberFormat] = useUiSetting$<string>(DEFAULT_NUMBER_FORMAT);
+const SignalsUtilityBarComponent: React.FC<SignalsUtilityBarProps> = ({
```

Comment:
Same clarification as above -- why `React.FC` then wrapped with `React.memo`? Also, both `SignalsUtilityBarComponent` and `SignalsUtilityBar` are missing `displayName` and no linter error is shown.